drivers/start_date_dq/app/controllers/start_date_dq/warehouse_reports/reports_controller.rb

```diff
@@ -0,0 +1,60 @@
+###
+# Copyright 2016 - 2022 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module StartDateDq::WarehouseReports
+  class ReportsController < ApplicationController
+    include WarehouseReportAuthorization
+    include ArelHelper
+    include BaseFilters
+    include Filter::FilterScopes
+
+    before_action :set_report
+    before_action :set_title
+
+    def index
+      respond_to do |format|
+        format.html do
+          if params[:filter].present?
+            data = @report.data
+            @pagy, @enrollments = pagy(data, items: 50)
+          end
```

Comment:
This could be a huge report if someone decides to run it against a large date range with all projects. In these cases, do we ever require filtering to a certain date range or filtering by project or project type?

Reply:
The filter object will limit them to a year date range automatically, so size shouldn't be an issue, in 90% of the scenarios.  There may be 10k+ records, but that would make the report fairly useless, so hopefully people are looking at a smaller sub-set.  Actual use may necessitate a different format or approach, but we'll pivot based on feedback if necessary.